gettsim/config.py

```diff
@@ -7,10 +7,10 @@
 GEP_1_CHARACTER_LIMIT = 17
 
 INTERNAL_FUNCTION_FILES = [
```

Comment:
`INTERNAL_FUNCTION_LOCATIONS` ? Most of these are directories, not files.

Reply:
Renaming to `PATHS_TO_INTERNAL_FUNCTIONS`?

Reply:
Fine with me. Maybe a little note that this will take all modules in a directory or a single module?